saga/sagaState.go

```diff
@@ -478,3 +478,47 @@ func makeSagaState(sagaId string, job []byte) (*SagaState, error) {
 
 	return state, nil
 }
+
+/*
+ * Custom ToString function for SagaState
+ */
+func (state *SagaState) String() string {
+
+	fmtString := "{ SagaId: %v, " +
+		"SagaAborted: %v, " +
+		"SagaCompleted: %v, " +
+		"Tasks: [ "
+
+	for _, id := range state.GetTaskIds() {
+
+		taskState := ""
+
+		if state.IsTaskStarted(id) {
+			taskState += "Started|"
+		}
+		if state.IsTaskCompleted(id) {
+			taskState += "Completed|"
+		}
+		if state.IsCompTaskStarted(id) {
+			taskState += "CompTaskStarted|"
+		}
+		if state.IsCompTaskCompleted(id) {
+			taskState += "CompTaskCompleted|"
+		}
+
+		if len(taskState) >= 3 {
```

Comment:
Removing trailing '|' it should be 1, it was 3 because I had some spaces when playing around with formatting.